src/devtools/WebAudioEventObserver.js

```diff
@@ -27,13 +27,10 @@ export class WebAudioEventObserver extends Observer {
       const onEvent = (debuggeeId, method, params) => {
         onNext({method, params});
       };
-      const onDetach = () => {
-        chrome.debugger.attach({tabId}, debuggerVersion, () => {
-          chrome.debugger.sendCommand(
-            {tabId},
-            ChromeDebuggerWebAudioDomain.Methods.enable,
-          );
-        });
+      const onDetach = ({tabId}, detachReason) => {
+        // TODO: Show a warning if the DevTools are still open and allow the
+        // user to re-attach manually, e.g. by pressing a button.
+        // See: https://developer.chrome.com/docs/extensions/reference/debugger/#type-DetachReason
```

Comment:
So the "Reattach" button is not implemented in this PR. Right?

Reply:
Not too difficult, but I prefer to add the re-attach button only after TypeScript and RxJS code health improvements have been merged.